src/nidcpower/metadata/config.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
-# This file is generated from NI-DCPower API metadata version 20.6.0d7
+# This file is generated from NI-DCPower API metadata version 20.7.0d7
 config = {
-    'api_version': '20.6.0d7',
+    'api_version': '20.7.0d7',
```

Comment:
Can't you use 20.6.0f0?

Reply:
Yes. This is a mistake. I'll fix it.

Reply:
To fix this the right way, we have cherry pick some metadata fixes we did and build a 20.6.0f1.

Reply:
Not worth it. Use 20.7.0d7 or whatever it is you need.